image/postgres/rhtap.Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as builder
+
+COPY . /stackrox
+
+FROM registry.redhat.io/rhel8/postgresql-13 as final
+
+USER root
+
+LABEL \
+    com.redhat.component="rhacs-central-db-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="Central DB for RHACS" \
+    io.k8s.description="Central DB for RHACS" \
```

Comment:
Is there an authoritative source for these descriptions, and tags? 
This information seems to be propagated to the catalog, e.g. for 4.2.2: https://catalog.redhat.com/software/containers/advanced-cluster-security/rhacs-central-db-rhel8/635face27140d313bbc4afbd?architecture=amd64&image=653112d59841e831b5558a38&container-tabs=technical-information .

Reply:
I did not know about that feature. I don't know the answer. Would suggest to ask in `pyxis` GChat.

Reply:
The required labels are explained [here](https://enterprisecontract.dev/docs/rhtap-policy/index.html#_required_oci_image_labels). 
Their values match what we have defined on midstream today. 
I created another reconcile ticket for this: https://issues.redhat.com/browse/ROX-20839.

Reply:
Required labels for Enterprise Contract may overlap with labels used by Red Hat Catalog, but I highly doubt these are the same sets (or that the catalog ones are a subset of EC labels).